Documentation/DocuBlocks/Rest/Graph/general_graph_create_http_examples.md

```diff
@@ -45,6 +45,8 @@ Cannot be modified later.
 
 @RESTSTRUCT{replicationFactor,post_api_gharial_create_opts,integer,required,}
```

Comment:
Now: Integer **OR** String ...

Reply:
Good point. The DocuBlock system does not support this as far as I can tell from the Python code which generates Swagger 😞 I think our options are to remove the type information or leave it as integer and write in the description that it also accepts the "satellite" string...

Reply:
Will remain as _integer_ for now, but I added something to the description to inform users that it can also be "satellite".